components/Snackbar/src/MDCSnackbarManager.m

```diff
@@ -562,6 +570,133 @@ + (void)resumeMessagesWithToken:(id<MDCSnackbarSuspensionToken>)inToken {
   [self handleInvalidatedIdentifier:token.identifier forCategory:token.category];
 }
 
+#pragma mark - Styling
+
++ (void)setSnackbarMessageViewBackgroundColor:(UIColor *)snackbarMessageViewBackgroundColor {
+  if (snackbarMessageViewBackgroundColor != _snackbarMessageViewBackgroundColor) {
+    _snackbarMessageViewBackgroundColor = snackbarMessageViewBackgroundColor;
+    if (_applyStylingOnCurrentSnackbar) {
+      dispatch_async(dispatch_get_main_queue(), ^{
+        MDCSnackbarManagerInternal *manager = [MDCSnackbarManagerInternal sharedInstance];
+        [manager.currentSnackbar
+            setSnackbarMessageViewBackgroundColor:snackbarMessageViewBackgroundColor];
+      });
+    }
+  }
+}
+
++ (UIColor *)snackbarMessageViewBackgroundColor {
+  return _snackbarMessageViewBackgroundColor;
+}
+
++ (void)setSnackbarMessageViewShadowColor:(UIColor *)snackbarMessageViewShadowColor {
+  if (snackbarMessageViewShadowColor != _snackbarMessageViewShadowColor) {
+    _snackbarMessageViewShadowColor = snackbarMessageViewShadowColor;
+    if (_applyStylingOnCurrentSnackbar) {
+      dispatch_async(dispatch_get_main_queue(), ^{
+        MDCSnackbarManagerInternal *manager = [MDCSnackbarManagerInternal sharedInstance];
+        [manager.currentSnackbar setSnackbarMessageViewShadowColor:snackbarMessageViewShadowColor];
+      });
+    }
+  }
+}
+
++ (UIColor *)snackbarMessageViewShadowColor {
+  return _snackbarMessageViewShadowColor;
+}
+
++ (void)setMessageTextColor:(UIColor *)messageTextColor {
+  if (messageTextColor != _messageTextColor) {
+    _messageTextColor = messageTextColor;
+    if (_applyStylingOnCurrentSnackbar) {
+      dispatch_async(dispatch_get_main_queue(), ^{
+        MDCSnackbarManagerInternal *manager = [MDCSnackbarManagerInternal sharedInstance];
+        [manager.currentSnackbar setMessageTextColor:messageTextColor];
+      });
+    }
+  }
+}
+
++ (UIColor *)messageTextColor {
+  return _messageTextColor;
+}
+
++ (void)setMessageFont:(UIFont *)messageFont {
+  if (messageFont != _messageFont) {
+    _messageFont = messageFont;
+    if (_applyStylingOnCurrentSnackbar) {
+      dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Is it worth checking whether we're on the main queue before dispatching asynchronously?

Reply:
done